client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -124,6 +158,13 @@ const get_default_state_from_props = (props: _DisplayTableProps) => {
     .pickBy((col) => col.is_searchable)
     .mapValues(() => "")
     .value();
+  const t0 = performance.now();
+  const filter_options_by_column = get_default_filter_options_by_column(
+    data,
+    col_configs_with_defaults
+  );
+  const t1 = performance.now();
+  console.log(t1 - t0);
```

Comment:
So how was the performance on the bigger RPB tables? Fine to drop the memoization?

Assume you meant to clean this up before flagging for review, haha.

Reply:
Haha oops, lodash native caching without resolver is slightly faster but still fast enough